### PR TITLE
Governance page

### PR DESCRIPTION
**Pull request by garbas**
This page was inspired by [Goverance](https://www.rust-lang.org/governance) page on Rust website and replaced/consumed existing Security item from the top navigation bar.

- [x] RFC Steering Committee needs to be update the page with correct info
- [x] Security team needs to update the page with correct info
- [x] Add NixOS release team
- [x] Infrastructure team
- [x] nixcon team
- [x] Add redirects from removed pages to /governance.html

Have I missed any other team?

Those of you reviewing this please try to ignore the design and focus on content.

### CODE REVIEW

**Comment by samueldr** [comment]
The removed pages will need a redirect.

 * [ ] Add missing redirects

**Comment by garbas** [comment]
Ready for final review

File: governance.tt [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+<div class="row-fluid">
+  <div class="span8">
+    <div class="hero-unit">
+      <h2>RFC Process</h2>
+      <p>The "RFC" (request for comments) process is intended to provide a
+         consistent and controlled path for new features to enter the Nix language,
+         packages and OS, so that all community can be confident about the
+         direction the ecosystem is evolving in.
+      </p>
+      <a class="btn btn-block btn-large btn-success">Learn more</a>
```
Comment (edolstra) [inline governance.tt:16]: This needs a link target.

File: governance.tt [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  </div>
+</div>
+
+<h2>Teams</h2>
+
+<div class="row-fluid">
+  <div class="span4">
+    <h3>RFC Steering Committee</h3>
+    <p>Responsible for forming an RFC Shepherd team from the available
+       nominations on each RFC. RFC Shepherd team is then responsible for
```
Comment (edolstra) [inline governance.tt:50]: "The RFC Shepherd team..." or maybe ", which is then responsible..."

File: governance.tt [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    <p>Security is a priority for the NixOS community. In addition to
+       community-sourced developments, NixOS has a dedicated team, to look into
+       privately reported security issues.
+    </p>
+    <a class="btn btn-block btn-success"
+       href="[%root%]/teams/security.html">Members &amp; Contacts</a>
+  </div>
+  <div class="span4">
+    <h3>Infrastructure Team</h3>
+    <p>The responsibility of the team is to provide infrastructure for the Nix and
```
Comment (edolstra) [inline governance.tt:67]: "of this team"

File: governance.tt [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    <a class="btn btn-block btn-success"
+       href="[%root%]/teams/infrastructure.html">Members &amp; Contacts</a>
+  </div>
+</div>
+
+<div class="row-fluid">
+  <div class="span4">
+    <h3>NixOS Release Team</h3>
+    <p>A team that twice a year manages a release of NixOS. It is responsible
+       whole release process from setting the roadmap to uploading the
```
Comment (edolstra) [inline governance.tt:79]: "responsible for the entire release process..."

File: governance.tt [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+       artifacts.
+    </p>
+    <a class="btn btn-block btn-success"
+       href="[%root%]/teams/nixos_release.html">Members &amp; Contacts</a>
+  </div>
+  <div class="span4">
+    <h3>NixCon Team</h3>
+    <p>A team that helps organize NixCon, a conference, where Nix and NixOS
```
Comment (edolstra) [inline governance.tt:87]: "a conference where"

Or maybe "a conference for the Nix and NixOS community".

File: governance.tt [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+       related technologies are discussed.
+    </p>
+    <a class="btn btn-block btn-success"
+       href="[%root%]/teams/nixcon.html">Members &amp; Contacts</a>
+  </div>
+  <div class="span4">
+    <h3>Marketing Team</h3>
+    <p>Main responsibility is this website and the content on it. Apart from
```
Comment (edolstra) [inline governance.tt:95]: "Its main responsibility"

File: governance.tt [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+       that team looks for ways to help and improve general communication of
```
Comment (edolstra) [inline governance.tt:96]: "the team"

**Comment by davidak** [comment]
This already looks great.

I actually had the idea to create such a site last year (https://github.com/NixOS/nixos-homepage/issues/284):

>We should describe the organization structure on the website, like the RFC Steering Committee, Nixpkgs Maintainers (New GitHub group) etc. [...] so we probably need a new page. Maybe the Foundation could be just one section on that page.

"Nixpkgs Maintainers" is currently missing, but they are an important part of the ecosystem.

Until we can get a list of maintainers from ElasticSearch and have a good description what these people do, we can just add it as section with title "Nixpkgs Maintainers" and text "The package maintainers keep the packages up-to-date and fix reported problems." without further link...

I think that is better than not mentioning them.

**Comment by worldofpeace** [comment]
@davidak I think @garbas mentioned (to me in private) this would go in the community page.

File: teams/nixos_release.tt [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  <li>create a roadmap</li>
+  <li>release in cooperation with Eelco Dolstra</li>
+  <li>decide which bug fixes, features, etc... get back ported after a release</li>
+</ul>
+
+<h2>Release process</h2>
+
+More about it you can
+<a href="https://nixos.org/nixos/manual/index.html#release-managers">
+  read in the manual</a>
```
Comment (worldofpeace) [inline teams/nixos_release.tt:39]: This reads funny to me. How about, more information can be found in the nixos manual?

File: teams/nixos_release.tt [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+  introduced to their role, making it easier to pass on knowledge and
+  experience.
+</p>
+
+<h2>Members</h2>
+
+<ul>
+  <li><a href="https://github.com/worldofpeace">@worldofpeace</a></li>
+  <li><a href="https://github.com/disassembler">@disassembler</a></li>
+</ul>
```
Comment (worldofpeace) [inline teams/nixos_release.tt:23]: I think it's important to specify for what release the manager has served primarily?
So I'm (20.03) and Samuel is (19.09).

**Comment by garbas** [comment]
on some pages I call people by full name somewhere by irc nick somewhere else by github handle. I guess I should choose one :)

**Comment by worldofpeace** [comment]
@garbas I think the community would understand full name and github handle in parenthesis.

**Comment by worldofpeace** [comment]
I do know of at least two people in the community that want to remain semi-anonymous, so in that case you should just use their GitHub. (I believe one of them is in the webpage actually)

**Comment by davidak** [comment]
>@davidak I think @garbas mentioned (to me in private) this would go in the community page.

yes, that makes sense. since the package maintainers are normal community members and there is no team that governs them. we might want to establish a packaging working group later to coordinate getting maintainers for abandoned packages, define workflow and responsibility of maintainers etc

**Comment by worldofpeace** [comment]
> > @davidak I think @garbas mentioned (to me in private) this would go in the community page.
> 
> yes, that makes sense. since the package maintainers are normal community members and there is no team that governs them. we might want to establish a packaging working group later to coordinate getting maintainers for abandoned packages, define workflow and responsibility of maintainers etc

Yep, I agree with you on all points.

**Comment by ryantm** [comment]
Another official source of governance is Discourse admins and moderators:

Admins:
ryantm
edolstra
grahamc
zimbatm

Moderators:
mic92

**Comment by ryantm** [comment]
I added a PR for it https://github.com/NixOS/nixos-homepage/pull/378

**Comment by ryantm** [comment]
Sorry, I was going to use the CI to test it first, it looks like it isn't good yet.

**Comment by ryantm** [comment]
okay #379 is better now (I actually tested it locally)

 cc @garbas

**Comment by garbas** [comment]
No worries. Thank you a ton for adding it.

**Comment by garbas** [comment]
> I do know of at least two people in the community that want to remain semi-anonymous, so in that case you should just use their GitHub. (I believe one of them is in the webpage actually)

I'll use what is published on github and provide link to github account

**Pull request merged**